packages/wonder-blocks-data/components/track-data.md

```diff
@@ -1,3 +1,208 @@
 The `TrackData` component is a server-side only component. It should be used as
 a parent to the components whose data requests you want to fulfill during
 server-side rendering.
+
+#### Client-side behavior
+
+If used outside of server-side mode (as set using `Server.setServerSide`), this
+component will throw, as demonstrated below.
+
+```jsx
+import {Body, BodyMonospace} from "@khanacademy/wonder-blocks-typography";
+import {Server, View} from "@khanacademy/wonder-blocks-core";
+import {TrackData} from "@khanacademy/wonder-blocks-data";
+
+class ErrorBoundary extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    static getDerivedStateFromError(error) {
+        return {error: error.message};
+    }
+
+    render() {
+        if (jest) {
```

Comment:
This fails in the prod site, we'll probably need to do: `typeof jest !== "undefined"`:

https://deploy-preview-626--wonder-blocks.netlify.com/#section-data

Reply:
Oh darn! Thanks. This was the very last thing I added too, and I neglected to check that. Thanks for taking a look. I'll update and test it.